README.md

```diff
@@ -1,49 +1,103 @@
 ```
 Since the migrator executes a destructive operation, it first prompts you to confirm. Use the `--non-interactive` option to avoid the prompt and begin the migration immediately.
 
+You should see output like below on the CLI from a successful run of the migrator.
+<details>
+<summary>Command output</summary>
+
+```
+Migrate \\.\PhysicalDrive0 with image .\balena-flasher-dev.img
+Require 42991616 (41.00 MB) for boot partition
+Require 3977248768 (3793.00 MB) for rootA partition
+Found 1048576 (1.00 MB) not allocated on disk \\.\PhysicalDrive0
+Shrink partition C by 4020240384 (3834.00 MB)
+Create flasherBootPartition
+Created new partition for boot at offset 49109008384 with size 42991616
+Create flasherRootAPartition
+Created new partition for data at offset 49152000000 with size 3977248768
+Copy flasherBootPartition from image to disk
+read: {"position":46137345,"bytes":41943041,"speed":655361128.9081677,"averageSpeed":655360015.625}
+write: {"position":41943041,"bytes":41943041,"speed":645276985.799303,"averageSpeed":645277553.8461539}
+Copy flasherRootAPartition from image to disk
+read: {"position":281018368,"bytes":234881024,"speed":939524096,"averageSpeed":939524096}
+write: {"position":232783872,"bytes":232783872,"speed":927426052.665908,"averageSpeed":927425784.8605578}
+read: {"position":422576128,"bytes":376438784,"speed":892862464,"averageSpeed":752877568}
+write: {"position":375390208,"bytes":375390208,"speed":882800964.0826695,"averageSpeed":749281852.2954092}
+...
+read: {"position":4022337537,"bytes":3976200193,"speed":692391510.844067,"averageSpeed":700652016.3876652}
+write: {"position":3976200193,"bytes":3976200193,"speed":692235395.8367282,"averageSpeed":700281823.3532934}
+Mount Windows boot partition and copy grub bootloader from image
+Cleared up mount M: for EFI
+Copying: /EFI/BOOT/BOOTX64.EFI 	~=>	 M:\EFI\Boot\BOOTX64.EFI
+Copying: /EFI/BOOT/GRUB.CFG 	~=>	 M:\EFI\Boot\GRUB.CFG
+Copying: /EFI/BOOT/GRUBENV 	~=>	 M:\EFI\Boot\GRUBENV
+Copying: /EFI/BOOT/grub_extraenv 	~=>	 M:\EFI\Boot\grub_extraenv
+Copied grub bootloader files
+Set boot file
+Boot file set. The operation completed successfully.
+
+Migration complete, about to reboot
+```
+
+</details>
+
+
+## Development
+Migrator provides a command line application to run Etcher SDK's [migrator function](https://github.com/balena-io-modules/etcher-sdk/tree/master/lib/migrator). It uses [oclif](https://oclif.io) and [pkg](https://www.npmjs.com/package/pkg) to provide the command line executable.
+
+To build the migrator, clone this repository, install the node modules and build it as shown below. Like the application itself, presently it builds only on Windows.
+
+```
+> git clone https://github.com/balena-os/migrator.git
+> cd migrator
+> npm install
+> npm run build
+> del node_modules\pkg\dictionary\drivelist.js
+> npm run pkg
+```
+These commands generate a single-file executable `migrator.exe` in the `dist` directory.
```

Comment:
I miss a list with future work that specifies that configuration migration and user data migration is not currenty supported but is on the roadmap.

Reply:
How about if we link to the roadmap [page](https://balena.fider.io/posts/2/provide-tool-to-onboard-migrate-devices-already-deployed-in-the-field) for the migrator so it's all in one place?

Reply:
Added link to roadmap page.